CollectorUtils/scripts/maintain_attachments.py

```diff
@@ -33,54 +33,55 @@ def enable_copy_attachments(input_fc, output_fc):
         input_attachment_table = desc.Path.split('.')[0] + '.gdb\\' + desc.Name + '__ATTACH'
                             
         if not arcpy.Exists(input_attachment_table):
-            arcpy.AddError("Unable to locate the attachment table for the input feature class.")
-            return
+            arcpy.AddMessage("Unable to locate the attachment table for the input feature class.")
 
-    # Enable Attachments
-    arcpy.AddMessage("Enabling Attachments")
-    arcpy.EnableAttachments_management(output_fc)
-    arcpy.AddMessage("Enabled Attachments")
+    if arcpy.Exists(input_attachment_table):
```

Comment:
You don't need this check. Your already checking for   "if not arcpy.Exists(input_attachment_table):"

Reply:
@nitinbandi the purpose for that check is to only enable attachments on the output FC if the input FC had attachments to begin with. Therefore if the ATTACH table exists, it will enter that block and enable the attachments.

Reply:
@Kevin-GIS You can refactor to use if else

if not arcpy.Exists(input_attachment_table):
   ....
else:
 ... ..

Reply:
@niti5425-zz 

I made that change and tested it and it looks good. If you feel it's good, I can merge it.